internal/executor/shell.go

```diff
@@ -167,6 +167,13 @@ func NewShellCommands(
 	cmd.Stderr = sc.piper.FileProxy()
 	cmd.Stdout = sc.piper.FileProxy()
 
+	if custom_env != nil {
+		if _, ok := custom_env.Lookup("CIRRUS_AGENT_EXPOSE_SCRIPTS_OUTPUTS"); ok {
+			cmd.Stderr = io.MultiWriter(cmd.Stderr, sc.piper.FileProxy())
+			cmd.Stdout = io.MultiWriter(cmd.Stdout, sc.piper.FileProxy())
```

Comment:
```suggestion
			cmd.Stderr = io.MultiWriter(os.Stderr, sc.piper.FileProxy())
			cmd.Stdout = io.MultiWriter(os.Stdout, sc.piper.FileProxy())
```

Reply:
First bug that Copilot almost introduced! Good catch!